tasks/main.yml

```diff
@@ -1,15 +1,18 @@
 ---
 - name: OS specific setup
   include: include: "setup-{{ ansible_os_family }}.yml"
+  when: docker_install
```

Comment:
I don't understand a reason to add skip_install to the role what is intended only to install